Makefile

```diff
@@ -106,6 +106,25 @@ manylinux-x86.test: manylinux-x86
 	$(BIN)/dockcross-manylinux-x86 /opt/python/cp35-cp35m/bin/python test/run.py
 
 #
+# Xtensa esp8266
```

Comment:
Instead of creating this custom target, add to `STANDARD_IMAGES` at the top of the Makefile.

Reply:
A fresh rebase on master may be required.